### PR TITLE
Data release 20240613

### PR DESCRIPTION
**Pull request by hareesh-ms**

### CODE REVIEW

File: internal/server/v0/placestatvar/golden/get_place_stat_vars/california.json [hunk 1]
```diff
@@ -35135,7 +35131,6 @@
         "dc/0hry4lccfhcgf",
         "dc/0hsbkjer9x9jf",
         "dc/0hslbewgd1h6c",
-        "dc/0hsryns4ejntb",
```
Comment (hareesh-ms) [inline internal/server/v0/placestatvar/golden/get_place_stat_vars/california.json:35138]: These are variables generated by the Sahie import poverty line changes which was reverted later to use the existing variables as part of CL#635471476. 

Series with the human readable SV is present in prod : https://screenshot.googleplex.com/5A34zFCkJVwXBNL.
Deleted :  https://screenshot.googleplex.com/7xtQsGFmrYrmnqe

After reverting, the import was added to frequent and rebuilt but the  us census cache was not rebuilt. Hence the new dc/ variables generated were part of the last release which is getting deleted now.

File: internal/server/v1/observations/golden/bulk_point_linked/all_FRA_AA2_2016.json [hunk 2]
```diff
@@ -33,21 +33,6 @@
             }
           ]
         },
-        {
-          "entity": "nuts/FR10",
```
Comment (hareesh-ms) [inline internal/server/v1/observations/golden/bulk_point_linked/all_FRA_AA2_2016.json:37]: Related to the nuts to Admin Area direct mapping revert ( CL#638945277)

File: internal/server/v1/observations/golden/bulk_point_linked/preferred_FRA_AA2_2016.json [hunk 3]
```diff
@@ -33,16 +33,6 @@
             }
           ]
         },
-        {
-          "entity": "nuts/FR10",
```
Comment (ajaits) [inline internal/server/v1/observations/golden/bulk_point_linked/preferred_FRA_AA2_2016.json:37]: Due to nuts places AA<N> reverted in cl/638945277

**Review by ajaits — APPROVED** [review]

File: internal/server/v1/variables/golden/bulk_variables/california.json [hunk 4]
```diff
@@ -43731,7 +43731,6 @@
         "dc/031mxslrxvjv8",
         "dc/03288qhhv2df4",
         "dc/033zp3qb7s7j6",
-        "dc/034hcwptwt6d5",
```
Comment (pradh) [inline internal/server/v1/variables/golden/bulk_variables/california.json:43734]: Any ideas on this? https://datacommons.org/browser/dc/034hcwptwt6d5

**Review by pradh — APPROVED** [review]

File: internal/server/v1/variables/golden/bulk_variables/california.json (hunk 4 above)
Comment (hareesh-ms) [reply internal/server/v1/variables/golden/bulk_variables/california.json:43734]: The old SV https://datacommons.org/browser/Count_Person_40To64Years_RatioToPovertyLineUpto1.38_HispanicOrLatino will be retained.

**Pull request merged**